lib/logatron/logatron.rb

```diff
@@ -72,5 +74,14 @@ def msg_id
     def msg_id=(id)
       Logatron::Contexts.msg_id = id
     end
+
+    private
+
+    def exception_message(e, additional)
+      info = additional.is_a?(Hash) ? additional.map { |(k, v)| "#{k}=>#{v}"}.join(', ') : additional
+      info_msg = info.nil? || info.empty? ? '' : "; MORE_INFO( #{info} )"
+      backtrace = configuration.backtrace_cleaner.clean(e.backtrace).join(' -> ')
+      "#{e.class} #{e.message}#{info_msg} -> #{backtrace}"
```

Comment:
The info_msg is optional, so shift it to the end of the log entry.

I think I'd prefer the format:

"#{e.class} #{e.message} -> #{backtrace} -> #{info_msg}"